meshtastic/interfaces/ble/client.py

```
@@ -180,6 +193,29 @@ def _sync_address_from_bleak(self) -> None:
+        if bleak_client is None:
+            raise self.BLEError(error_message)
+        return bleak_client
+
+    def _run_transport_operation(
+        self,
+        *,
+        error_message: str,
+        operation: Callable[[BleakRootClient], Coroutine[Any, Any, Any]],
+        timeout: float | None = None,
+        sync_address: bool = False,
+    ) -> Any:
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_run_transport_operation`

(ANN401)